client/src/components/overview/AddToCart.jsx

```diff
@@ -1 +1,68 @@
-import React from 'react';
\ No newline at end of file
+import React, { useState } from 'react';
+import { Container, Button, Row, Form, FormGroup, FormControl, Col } from 'react-bootstrap';
+import { useSelector } from 'react-redux';
+import API_KEY from '../../config/config'
+import axios from 'axios';
+
+function AddToCart(props) {
+  const widget = 'add-to-cart'
+  const skusInfo = useSelector((state) => state.style.skus);
+  const skus = Object.keys(skusInfo);
+  const sizes = [];
+  Object.values(skusInfo).forEach(sku => {
+    sizes.push(sku.size);
+  });
+  const sizeSelector = sizes.map((size, index) => <option  key={index} value={skus[index]}>{size}</option>);
+  const [size, setSize] = useState('');
```

Comment:
I think this works great for now, bus should we be relying at all on vanilla React state if we are using the Redux store for all of our state calls? I think this works perfectly as is for testing and initial implementation, but we might want to move all of our state to the store in a future refactor. Let me know your thoughts!